.github/workflows/ci-macvim.yaml

```diff
@@ -38,18 +41,27 @@ jobs:
         - os: macos-10.15
           xcode: 11.7
         - os: macos-10.15
-          publish: true
         - os: macos-11.0
+          publish: true
```

Comment:
current macos is 11.1 -- is that version available in ci?

Reply:
No: https://docs.github.com/en/free-pro-team@latest/actions/reference/specifications-for-github-hosted-runners#supported-runners-and-hardware-resources

Why does it matter though? I am not aware of any breaking issues in 11.0 that would prevent CI from running. Most of the work is done by Xcode anyway

Reply:
I guess it doesn't matter 👍